src/api/spec/features/webui/users/beta_features_spec.rb

```diff
@@ -87,7 +89,10 @@
     end
 
     context 'when enabling a beta feature which is already enabled' do
-      let!(:disabled_beta_feature) { create(:disabled_beta_feature, name: beta_feature, user: user) }
+      let!(:disabled_beta_feature) do
+        Flipper::Adapters::ActiveRecord::Feature.create(key: beta_feature)
+        create(:disabled_beta_feature, name: beta_feature, user: user)
```

Comment:
Sounds like `Flipper::Adapters::ActiveRecord::Feature.create` should go into the factory?

Reply:
@hennevogel I've moved feature creation to the factory but there is one case where it is not using that factory.